README.md

```diff
@@ -53,6 +58,10 @@ We're all on Mozilla's IRC in the [#devtools-html][irc-devtools-html] channel on
 
 [getting-started]:./docs/getting-setup.md
 [contributing]:./CONTRIBUTING.md
+[getting-started-screencast]:/docs/videos#getting-started
+[up-for-grabs]:https://github.com/devtools-html/debugger.html/issues?label=???
```

Comment:
Use this: https://github.com/devtools-html/debugger.html/issues?q=is%3Aissue+is%3Aopen+label%3A%22up+for+grabs%22